src/managers/BaseManager.ts

```diff
@@ -64,21 +64,18 @@ export abstract class BaseManager {
     try {
       isKeySuccessfullyLocked = await this.storage.lockKey(context.key);
     } catch (keyLockError: unknown) {
-      this.logger.error(
+      this.logger.info(
```

Comment:
please provide more reasons for this and below logging level changes

Reply:
Sure.
A failed attempt to acquire a lock on the key does not lead to a fail of operation as whole.

Reply:
I mean that it's better to make separate commit with according commit message. This change will appear in changelog.

Reply:
done